WordPress/Classes/Models/WPAccount.m

```diff
@@ -52,6 +52,10 @@ - (void)didTurnIntoFault
 
 - (NSString *)password
 {
+    if (self.isWpcom) {
+        return [NSString string];
```

Comment:
I took the empty string road, there, mostly, [to avoid this kind of scenario](https://github.com/wordpress-mobile/WordPress-iOS/blob/develop/WordPress/Classes/Networking/PostServiceRemoteXMLRPC.m#L129) (which would otherwise require us to check case by case).

However, on second thoughts, XMLRPC shouldn't be called for dotcom accounts. I'll make the arrangements so the password is *nil* (and will also add failsafes where needed).

Thank you Diego!